src/api/parser/src/data/post.js

```diff
@@ -123,7 +123,8 @@ class Post {
     // link is the URL of the post
     if (!article.link) missing.push('link');
     // guid is the unique identifier of the post
-    if (!article.guid) missing.push('guid');
+    // consider missing guid that is returned as an { '$': { isPermaLink: 'false' } } object.
+    if (!article.guid || article.guid.$) missing.push('guid');
```

Comment:
This is fine.  But follow-up question: could we substitute the `link` here instead, since the `guid` will often be the URL of the article?